sierra_adapter/sierra_items_to_dynamo/src/main/scala/uk/ac/wellcome/platform/sierra_items_to_dynamo/services/WindowExtractor.scala

```diff
@@ -1,44 +0,0 @@
-package uk.ac.wellcome.platform.sierra_items_to_dynamo.services
```

Comment:
Where did this go?

Reply:
Nowhere, we had two copies of it and now we have just one.